vsintegration/src/FSharp.Editor/LanguageService/LanguageService.fs

```diff
@@ -194,14 +194,15 @@ type internal FSharpWorkspaceServiceFactory [<Composition.ImportingConstructor>]
                             if enableLiveBuffers then
                                 workspace.WorkspaceChanged.Add(fun args ->
                                     if args.DocumentId <> null then
-                                        backgroundTask {
+                                        cancellableTask {
                                             let document = args.NewSolution.GetDocument(args.DocumentId)
 
                                             let! _, _, _, options =
                                                 document.GetFSharpCompilationOptionsAsync(nameof (workspace.WorkspaceChanged))
 
                                             do! checker.NotifyFileChanged(document.FilePath, options)
                                         }
+                                        |> CancellableTask.startAsTask CancellationToken.None
```

Comment:
Also - can be anyhow taken from the context?

Reply:
Which context? It's extension initialization code, there's no cancellation token available there.

Reply:
Yeah I see now, alright nrm.